packages/vhd-lib/src/Vhd/VhdSynthetic.integ.spec.js

```diff
@@ -0,0 +1,65 @@
+/* eslint-env jest */
+
+import rimraf from 'rimraf'
+import tmp from 'tmp'
+import { Disposable, pFromCallback } from 'promise-toolbox'
+import { getSyncedHandler } from '@xen-orchestra/fs'
+
+import { SECTOR_SIZE } from '../_constants'
+import { createRandomFile, convertFromRawToVhd } from '../tests/utils'
+import { openVhd } from '../../dist'
```

Comment:
Don't depend on the `dist` directory.

```suggestion
import { openVhd } from 'vhd-lib'
```